src/EnergyPlus/WindowAC.cc

```diff
@@ -721,27 +717,13 @@ namespace WindowAC {
         //       AUTHOR         Fred Buhl
         //       DATE WRITTEN   May 2000
         //       MODIFIED       Chandan Sharma, FSEC, March 2011: Added zone sys avail manager
-        //       RE-ENGINEERED  na
 
         // PURPOSE OF THIS SUBROUTINE:
         // This subroutine is for initializations of the Window AC Components.
 
         // METHODOLOGY EMPLOYED:
         // Uses the status flags to trigger initializations.
 
-        using DataZoneEquipment::CheckZoneEquipmentList;
-        using HVAC::SmallLoad;
-
-        int InNode;          // inlet node number in window AC loop
-        int OutNode;         // outlet node number in window AC loop
-        int InletNode;       // inlet node number for window AC WindACNum
-        int OutsideAirNode;  // outside air node number in window AC loop
-        int AirRelNode;      // relief air node number in window AC loop
-        Real64 RhoAir;       // air density at InNode
-        int Loop;            // loop counter
-        Real64 QToCoolSetPt; // sensible load to cooling setpoint (W)
-        Real64 NoCompOutput; // sensible load delivered with compressor off (W)
-
```

Comment:
[src/EnergyPlus/WindowAC.cc:735]:(style),[variableScope],The scope of the variable 'InNode' can be reduced.
[src/EnergyPlus/WindowAC.cc:736]:(style),[variableScope],The scope of the variable 'OutNode' can be reduced.
[src/EnergyPlus/WindowAC.cc:741]:(style),[variableScope],The scope of the variable 'Loop' can be reduced.